shortcodes/class.geojson-shortcode.php

```diff
@@ -75,7 +76,14 @@ protected function getHTML ($atts='', $content=null) {
                     layer = L.ajaxGeoJson(src, {
                         type: '<?php echo $class::$type; ?>',
                         style : layerStyle,
-                        onEachFeature : onEachFeature
+                        onEachFeature : onEachFeature,
+                        /* function to add custom icons when defined in feature.properties */
+                        pointToLayer: function(feature, latlng) {
+                            return feature.properties.iconUrl ? new L.Marker(latlng, {icon: L.icon({
+                                iconUrl: feature.properties.iconUrl,
+                                iconSize: [40, 40], // size of the icon
```

Comment:
Is this definitely the icon size for all icons? could this be settable as well?
Also could be useful to have the iconAnchor settable - although not necessary.

Reply:
You're right. I assumed custom icons would be 50x50 by default, but that's a bad assumption. I changed it to read a specified value from feature.properties.iconSize also and default to [25,41] which is the size used for the standard icon.